Code/autopkglib/MunkiInstallsItemsCreator.py

```diff
@@ -17,6 +17,7 @@
 
 import plistlib
 import subprocess
+from distutils.version import LooseVersion
```

Comment:
Use autopkglib's APLooseVersion instead of this. We will need to change the way LooseVersion comparisons work in the future because distutils.version is being removed in Py 3.12, so if you use the autopklib provided subclass, you/we won't have to refactor this in the future.

Reply:
Swapped out in https://github.com/autopkg/autopkg/pull/801/commits/de3640120aeff1f5b9bc7c602b0f8e795047fe69